### DISCUSSION TITLE
Next.js 13 + support fro menu items

### DISCUSSION THREAD

**Discussion by vitali-duckma** [ant-design/ant-design#39131] [Q&A]
Hi,  I started a project using antd with next.js 13 (latest), all seems ok but how to pass url inside the menu items?

**Comment by chunsch** [comment]
Can be added in label，for example:
```tsx
      <Menu
        items={[
          {
            key: '/',
            label: <Link href="/">Home</Link>
          },
          {
            key: '/blog',
            label: <Link href="/blog">Blog</Link>
          }
        ]}
      />
```

**Reply by vitali-duckma** [reply]
thanks a lot

**Reply by vitali-duckma** [reply]
hi, it doesn't work properly due too much padding ot menu item. you can try and see. because link must be fullwidth setted.

**Reply by chunsch** [reply]
Can you provide a demo?

**Reply by vitali-duckma** [reply]
> Can you provide a demo?

sure:
![Screenshot 2022-12-05 alle 08 59 42](https://user-images.githubusercontent.com/118166188/205584436-05be639f-d6b0-4072-a613-50a401bf7cf6.png)

**Reply by jake-nz** [reply]
For clarity, the problem @vitali-duckma is talking about is that the user must click on the text exactly. The whole menu item should be an `<a>` tag but only the text inside it is.
